src/Arcus.Messaging.Tests.Unit/ServiceBus/IServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Arcus.Messaging.Tests.Unit.ServiceBus
```

Comment:
Shouldn't we also assert that it registers the correct lookup?

Reply:
Hmm, how would you do that without making it an integration test?

Reply:
Check the type that was registered in the collection and how it looks it up, if possible.

Or use mocking to see which one was called how many times?

I don't know, just thinking out loud.

Reply:
Ah, now I see what you mean. I'll check that.